packages/imperative/src/doc/IImperativeConfig.ts

```diff
@@ -241,6 +250,14 @@ export interface IImperativeConfig {
      *
      * TODO: While the pluginHealthCheck property is validated, no Imperative command
      * currently calls the health check function.
+     *
+     * @deprecated
```

Comment:
The pluginHealthCheck property was defined years ago, but nothing in CLI/imperative ever implemented calling a plugin's healthcheck. One time that you might want to perform a healthcheck is right after the plugin has been installed. A plugin can now perform a healthcheck in the new postInstall function. Deprecating pluginHealthCheck gives us the opportunity to remove it in Zowe V3.

Your comment raises the legitimate question of whether the team thinks that a healthcheck could or should be implemented as a separate command in the future, something like 'zowe plugins healthcheck PluginName'. That would give users the opportunity to perform a healthcheck on-demand rather than just during installation. Since healthcheck has remained unimplemented for 4 years, it seemed to me that we are unlikely to ever implement it. That is why I took this opportunity to deprecate it. If the team expects that we might implement it in the foreseeable future, I would be willing to remove the deprecated tag.

I previously placed the removal of healthcheck into our spreadsheet of V3 work. If we choose to deprecate healthcheck, we do not have to record anything further.